<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: lightblue;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>N (1, 2,  3)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>R (0, 1)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" alignment="Qt::AlignRight">
     <widget class="QLineEdit" name="rValLine">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSlider" name="rValSlider">
      <property name="cursor">
       <cursorShape>ClosedHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>19</number>
      </property>
      <property name="sliderPosition">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>P (0, 2)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" alignment="Qt::AlignRight">
     <widget class="QLineEdit" name="pValLine">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" alignment="Qt::AlignRight">
     <widget class="QLineEdit" name="nValLine">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSlider" name="pValSlider">
      <property name="cursor">
       <cursorShape>ClosedHandCursor</cursorShape>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>39</number>
      </property>
      <property name="sliderPosition">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="nValSlider">
      <property name="cursor">
       <cursorShape>ClosedHandCursor</cursorShape>
      </property>
      <property name="toolTipDuration">
       <number>-4</number>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="pageStep">
       <number>3</number>
      </property>
      <property name="sliderPosition">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>lib/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
